governance/votes/Executive vote - March 9, 2022.md

```diff
@@ -0,0 +1,52 @@
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an out-of-schedule executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+This out-of-schedule executive proposal mitigates the post-shutdown flash loan vulnerability in the Maker Protocol as described by the Protocol Engineering Core Unit [here](https://forum.makerdao.com/t/post-shutdown-flash-loan-vulnerability-announcement-and-remediation-plan/13746). Note that there is no risk to Maker Protocol or any of its stakeholders so long as Emergency Shutdown has not been triggered. 
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Mitigate Post-Shutdown Flash Loan Vulnerability
+
+If this executive proposal passes, it will mitigate the post-shutdown flash loan vulnerability in the Maker Protocol as described by the Protocol Engineering Core Unit [here](https://forum.makerdao.com/t/post-shutdown-flash-loan-vulnerability-announcement-and-remediation-plan/13746). It does this by:
+* Giving a new module `flash-killer` authorization over the [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module), such that anyone can permissionlessly use `flash-killer` to disable the [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module) in the event of Emergency Shutdown being triggered.
+* The vulnerability is not exploitable until 3 days (currently) after Emergency Shutdown is triggered, meaning there is a 3-day window where any user can disable the [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module) using `flash-killer` before an exploit can take place.
+
+This is considered a temporary mitigation, and it is recommended that the issue be solved more permanently in the future via the replacement of the [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module).
```

Comment:
Maybe rephrase this as "PECU considers this to be a temporary mitigation...." to make it a bit clearer